### PR TITLE
Add gravity test

### PR DESCRIPTION
**Pull request by Karang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "jest --verbose",
-    "pretest": "npm run lint",
+    "pretest": "npm run lint && require-self",
```
Comment (rom1504) [inline package.json:11]: if we add require-self we need it in prepare too like https://github.com/PrismarineJS/mineflayer/blob/master/package.json#L12
so people cloning have it installed

Comment (Karang) [reply package.json:11]: done

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
...
+    "prepare": "require-self",
```
Comment (rom1504) [inline package.json:12]: Need the npm install require-self here too or npm publish will fail. (Yeah it's not great but there is no other way due to the absence of a "local-install" npm script)

Comment (Karang) [reply package.json]: done

**Pull request merged**